.github/workflows/publish.yaml

```diff
@@ -0,0 +1,47 @@
+---
+name: Publish
+
+on:
+  release:
+    types:
+      - released
+      - prereleased
+  workflow_dispatch: {}
+
+permissions:
+  contents: read
+  id-token: write
+  packages: write
+
+jobs:
+  Sync:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Sync Repositories
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{ secrets.GH_PERSONAL_ACCESS_TOKEN }}
+          script: |
+            const { data } = await github.rest.search.code({
+              q: 'Common Config org:beam-community path:.github/workflows/',
```

Comment:
We should be able to just query for the common-config.yaml file

Reply:
```suggestion
              q: 'org:beam-community path:.github/workflows/common-config.yaml',
```